payjoin/tests/integration.rs

```diff
@@ -679,7 +941,7 @@ mod integration {
             let receiver_utxos = receiver.list_unspent(None, None, None, None, None)?;
             assert_eq!(100, receiver_utxos.len(), "receiver doesn't have enough UTXOs");
             assert_eq!(
-                Amount::from_btc(3700.0)?, // 48*50.0 + 52*25.0 (halving occurs every 150 blocks)
+                Amount::from_btc(3650.0)?, // 50 (starting reciever blance) + 46*50.0 + 52*25.0 (halving occurs every 150 blocks)
```

Comment:
If you gotta document it I think it's better off written explicitly `Amount::from_btc(50 + 46*50.0 + 52*25.0)` and getting real fancy would be with constants `REWARD_H1 = 50`, `REWARD_H2 = 25`. Code semantics are harder to go stale than comments.